examples/simple_example/main.tf

```diff
@@ -14,17 +14,20 @@
  * limitations under the License.
  */
 
-provider "google" {
-  version = "~> 3.0.0"
-}
-
 module "kms" {
-  source     = "../.."
+  source  = "terraform-google-modules/kms/google"
```

Comment:
We use the examples for testing, so we should continue to keep these as `../..`

Reply:
fixed